neqo-http3/tests/connect_udp.rs

```diff
@@ -493,3 +496,176 @@ fn connect_udp_operation_on_fetch_stream() {
         Err(Error::InvalidStreamId)
     );
 }
+
+#[test]
+fn session_lifecycle_with_http_datagram_capsule() {
+    fixture_init();
+    neqo_common::log::init(None);
+
+    let conn_params = ConnectionParameters::default()
+        .pmtud(true)
+        .datagram_size(0)
+        .outgoing_datagram_queue(0)
+        .incoming_datagram_queue(0);
```

Comment:
I don't think setting the queue limits is necessary. `.datagram_size(0)` should be enough.